packages/babel/test/as-input-plugin.js

```diff
@@ -377,6 +377,7 @@ test('supports customizing the loader', async (t) => {
   const customBabelPlugin = createBabelInputPluginFactory(() => {
     return {
       config(cfg) {
+        t.true(typeof this === 'object', 'override config this context is rollup context');
```

Comment:
could u test smth more distinct? it's rather easy to regress based on this simple check

Reply:
@Andarist do you have an idea how to do that? Unfortunately I'm no expert in rollup, so I don't know wether I can do something like `instanceof` or similar.

Reply:
just maybe try to check some example property/methods that exist on this object?

Reply:
@Andarist alright, done.. Wanted to do this initially, but then I thought such checks could also be a bit shaky.
3b0e78b